src/constants.js

```diff
@@ -138,8 +138,8 @@ export const urls = {
     usage: "https://conda-forge.herokuapp.com/status-monitor/report/travis-ci",
   },
   versions: {
-    api: "https://raw.githubusercontent.com/regro/cf-graph-countyfair/master/status/version_status.json",
-    pr: "https://github.com/conda-forge/<NAME>-feedstock/blob/main/recipe/meta.yaml",
+    api: "https://raw.githubusercontent.com/regro/cf-graph-countyfair/master/status/version_status.v2.json",
+    pr: "https://github.com/conda-forge/<NAME>-feedstock/tree/main/recipe",
```

Comment:
Changed this so we are prepared for the recipe.yaml feedstocks too. Otherwise this will 404 and I don't think we have a quick way of checking whether `meta.yaml` or `recipe.yaml` is in place. So the recipe directory it is.